mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,68 @@
+import CommunitySlider from '@react-native-community/slider';
+import React from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    thumbTitleValue: string;
+    sliderValue: number;
+    onSliderValueChange: (value: number) => void;
+    onResetClickHandler: () => void;
+    containerStyle?: StyleProp<ViewStyle>;
+    minSliderValue?: number;
+    maxSliderValue?: number;
+    thumbTitleValueWidth?: number;
```

Comment:
```suggestion
type SliderProperties = SliderProps & {
    thumbTitleValue: string;
    containerStyle?: StyleProp<ViewStyle>;
```
Let's keep only custom props here

Reply:
Good morning, but it shall destroy abstact layer for the library or it's okey for us?

Reply:
What do you mean

Reply:
fix